ios/config.json

```diff
@@ -10,10 +10,12 @@
   "additionalLinkerFlags": [
     "-ObjC"
   ],
-  "userDefined": [
-    "onesignalAppID"
-  ],
   "plist": {
-    "onesignalAppID": "$(onesignal.onesignalAppID)"
+    "UIBackgroundModes": [
+      "remote-notification"
+    ]
+  },
+  "entitlements": {
```

Comment:
it should be the other way right? `app-environment` in plist and `UIBackgroundMode` in entitlements?